### DISCUSSION TITLE
Jupyter-Book Logo Use Question

### DISCUSSION THREAD

**Discussion by Alex-Lewandowski** [executablebooks/meta#1124] [Q&A] [answered]
I have been developing a JupyterLab server extension to allow navigation of a Jupyter-Book via its table of contents in a sidebar widget.

https://github.com/ASFOpenSARlab/jupyterlab-jupyterbook-navigation

I would like to use Jupyter-Books to organize content for classes and training events delivered in JupyterHub, where everyone works in JupyterLab. In that context, a Jupyter-Book doesn't need to be rendered at all. In JupyterLab, the organizational structure provided by the table of contents is needed, not a rendered static website. 

The extension allows us to clone a Jupyter-Book (rendered or not), navigate to it in the file browser, and interact with it via its table of contents. Notebooks and Markdown docs open in the JupyterLab workspace (MD is rendered), and links open in new browser tabs.

Would it be okay and appropriate to use the Jupyter-Book logo for the sidebar widget?

https://github.com/executablebooks/meta/assets/37909088/cde1b80c-733b-4870-a75f-23774f1ae93f

**Comment by choldgraf** [comment, marked as answer]
You're certainly welcome to use the logo in the UI provided by this extension!

You should also speak with @agoose77 who is interested in this idea as well.

**Comment by Alex-Lewandowski** [comment]
@choldgraf That's great; thank you for the go ahead!

@agoose77 I'd love to speak with you about the extension if you're interested. I noticed that you're at 2i2c. I was at the [2023 ICESat-2 Hackweek](https://icesat-2-2023.hackweek.io/), working in a 2i2c deployment ([CryoCloud](https://book.cryointhecloud.com/intro.html)), and a Jupyter-Book was used to organize the content. I thought it was a great way to organize everything, but the experience got a little bumpy when it came time to run the notebooks from within JupyterLab. I work at the [Alaska Satellite Facility](https://asf.alaska.edu/) on the  ([OpenScienceLab](https://opensciencelab.asf.alaska.edu/)) team, which is a collection of JupyterHubs, mostly used for teaching. That was my motivation for starting this project.

**Reply by agoose77** [reply]
@Alex-Lewandowski this looks very interesting! @choldgraf is right that I'm also keen for us to improve on the multi-document experience in `jupyterlab-myst`. In the long run, I plan to build-out the components used in MyST-CLI to provide this experience, but we need to work on a few key pieces to make that happen. 

Would you be interested in arranging a meeting at some point? It would be interesting to talk about your use case, and think about where we can share development / reduce the maintenance burden!

You can book a meeting into my calendar [here](https://calendar.app.google/V2PXsGbMF3LnYrDL9), if you're up for it!

**Reply by Alex-Lewandowski** [reply]
@agoose77 Thanks! I just booked a meeting and look forward to chatting next week!
